README.md

```diff
@@ -24,9 +27,107 @@ Other testing frameworks often support fixtures, which bracket a set of tests wi
+| `--update-release` RELEASE   | OSG release version (e.g. 3.2) to use when updating packages specified with -i.                                                                                                                                                            |
+| `--tarballs`                 | Test client tarballs instead of RPM-based installation.                                                                                                                                                                                    |
+| `--tarball-test-dir`         | The location of the tarball test files (if non-standard).                                                                                                                                                                                  |
+| `--no-print-test-name`       | Do not print test name before command output                                                                                                                                                                                               |
+| `--hostcert`                 | Create host cert                                                                                                                                                                                                                           |
+| `-T`, `--no-tests`           | Skip running the tests themselves. Useful for running/testing just the set-up and/or clean-up steps.                                                                                                                                       |
+| `-u`, `--test-user` USERNAME | Use the test user account with the given name. See also the `-a` and `-p` options.                                                                                                                                                         |
+| `-v`, `--verbose`            | Print the name of each test as it is run; generally a good idea.                                                                                                                                                                           |
+| `-h`, `--help`               | Print usage information and exit.                                                                                                                                                                                                          |
+| `--version`                  | Print the script version and exit.                                                                                                                                                                                                         |
+
+### Config file syntax ###
```

Comment:
Does these `###` actually do anything at the end of the line?

Reply:
Nope, it's just what `markdown-mode` does when I create a header < 2. I think it looks pretty :)